packages/perseus/src/widgets/label-image/label-image.tsx

```diff
@@ -310,21 +305,27 @@ export class LabelImage
     }
 
     getUserInput(): PerseusLabelImageUserInput {
-        const {markers} = this.props;
-        return {markers};
+        return {
+            markers: this.props.markers.map((marker) => ({
+                selected: marker.selected,
+                label: marker.label,
+            })),
+        };
```

Comment:
@handeyeco for visibility, Tamara and I re-wrote this function to build a return value by explicitly picking the pieces we need instead of simply return a value out of props. This help stop guarantee we're really only using values in the type and not depending elsewhere on keys that may get stripped when we do the Scoring Data stripping stuff.